ops/scripts/utility/check-env-hashes.sh

```diff
@@ -18,10 +18,14 @@ Help()
    echo
    echo "Syntax: ./ops/scripts/utility/check-env-hashes.sh [-l|h|r|e {hash}]"
    echo "options:"
-   echo "l     Check local branches."
+   echo "a     Database account name. Required unless using -e flag."
```

Comment:
I would honestly rather move away from the single digits in this fie to the more descriptive names in the other files. I am not sure why only this file contains the single char arguments.